src/components/RadioButton/RadioButton.tsx

```diff
@@ -1,5 +1,7 @@
 import React from 'react';
 import {useUniqueId} from '../../utilities/unique-id';
+import {useFeatures} from '../../utilities/features';
```

Comment:
Really petty nitpick: Can imports at a certain level be alphabetically sorted?

(A autofixable rule for this recently got added to eslint-plugin-import and IIRC other folks were also hoping for it so I expect it'll be added to the shopify eslint config at some point)